citations/citations-tool/tool/src/java/org/sakaiproject/citation/tool/CitationHelperAction.java

```diff
@@ -1402,6 +1402,18 @@ protected void captureDisplayName(ParameterParser params, SessionState state,
 		
 	}
 
+	protected void captureManagingLibrary(ParameterParser params, SessionState state,
```

Comment:
Cleaned it up by removing the 'Reading List' / 'Citation List' stuff and fixed some indentation.